src/Symfony/Component/Form/Test/TypeTestCase.php

```diff
@@ -31,7 +32,7 @@ protected function setUp(): void
     {
         parent::setUp();
 
-        $this->dispatcher = $this->createMock(EventDispatcherInterface::class);
+        $this->dispatcher = new EventDispatcher();
```

Comment:
Same here, the dispatcher is protected so someone could have added expectation on the dispatcher in his tests

Reply:
same here